### PR TITLE
Add Kibana to Telescope

### PR DESCRIPTION
**Pull request by c3ho**
## Issue This PR Addresses
Part of #1269

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
Submitting a new PR because previous one was reverted. When the previous one was merged, staging was failing to build due to #1261 I think. `docker-compose` command was failing because the env var for Kibana port could not be found.

I was thinking of Kibana ever since we added ElasticSearch, great to finally have it added. I had to also update our ElasticSearch image to 7.9.3 so it is compatible with the current version of Kibana.

To test:

1. make sure the extra lines from `env.example` are copied into your `.env` file
2. `docker-compose up redis elasticsearch kibana`
3. navigate to `http://localhost:5601`

You should see something like this afterwards

image

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: env.example [hunk 1]
```diff
@@ -59,6 +59,9 @@ ELASTIC_MAX_RESULTS=50
 # Delay to check connectivity with Elasticsearch in ms
 ELASTIC_DELAY_MS=10000
 
+# Kibana info
+KIBANA_PORT=5601
```
Comment (humphd) [inline env.example:63]: What if I want to run this on a different box/VM than my localhost locally?

**Review by humphd — APPROVED** [review]

File: env.example (hunk 1 above)
Comment (c3ho) [reply env.example:63]: Got it, I'll add `KIBANA_URL=` to the env.example

**Review by cindyledev — APPROVED** [review]
Tested this locally using Docker on Windows. This GUI is pretty neat :o

**Review by humphd — APPROVED** [review]
Code looks right to me.  I want sign-off from our docker experts too, so I'll flag them in review.

**Review by raygervais — APPROVED** [review]

**Comment by raygervais** [comment]
Hoping this resolves the previous issue in staging!

**Pull request merged**